docs/source/GETTING_STARTED.rst

```diff
@@ -99,7 +99,7 @@ You are now ready to start the Kiosk!
 
     docker run -e DOCKER_TAG=1.1.0 vanvalenlab/kiosk:1.1.0 | sudo bash
 
-.. note:: This command and the one that follows may need to be preceded by `sudo` depending on your permission settings.
+.. note:: This command and the one that follows may need to be preceded by `sudo` depending on your permission settings. This will require you to enter your password.
```

Comment:
I think this is overkill as sudo automatically asks for a password that you should know. But if you/Noah feels like it needs to be there I'm fine with it.

Reply:
I agree, I think its a bit overkill, but if we anticipate users to have little knowledge this should be helpful.